src/seams_input.cpp

```diff
@@ -87,13 +88,13 @@ int sinp::readXYZ(std::string filename,
     std::cout
         << "Fatal Error: The file does not exist or you gave the wrong path.\n";
     // Throw exception?
-    return 1;
+    return yCloud;
```

Comment:
```suggestion
        throw std::runtime_error("Wrong filepath");
```

Though this may not be a good idea (throwing errors would need to be handled more in the binding code).